packages/api/orm/migration/1638312144983-initial.ts

```diff
@@ -0,0 +1,142 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class initial1638312144983 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    const sql = `
+    CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+    
+    CREATE TYPE provisioning_status_enum AS ENUM (
+        'PENDING',
+        'IN_PROGRESS',
+        'COMPLETE',
+        'FAILED'
+    );
+    
+    CREATE TYPE csp_enum AS ENUM (
+        'CSP A',
+        'CSP B'
+    );
+    
+    CREATE TYPE file_scan_status_enum AS ENUM (
+        'PENDING',
+        'ACCEPTED',
+        'REJECTED'
+    );
+    
+    CREATE TYPE dod_component_enum AS ENUM (
+        'AIR_FORCE',
+        'ARMY',
+        'MARINE_CORPS',
+        'NAVY',
+        'SPACE_FORCE',
+        'COMBATANT_COMMAND',
+        'JOINT_STAFF',
+        'DAFA',
+        'OSD_PSAS',
+        'NSA'
+    );
+    
+    CREATE TABLE portfolio (
+        id uuid DEFAULT uuid_generate_v4() CONSTRAINT "pk_portfolio" PRIMARY KEY,
+        "createdAt" timestamp without time zone DEFAULT now() NOT NULL,
+        "updatedAt" timestamp without time zone DEFAULT now() NOT NULL,
+        "archivedAt" timestamp without time zone,
+        "provisioningStatus" provisioning_status_enum DEFAULT 'PENDING'::provisioning_status_enum NOT NULL,
+        administrators character varying[] DEFAULT '{}'::character varying[] NOT NULL,
+        contributors character varying[] DEFAULT '{}'::character varying[] NOT NULL,
+        "readOnlyOperators" character varying[] DEFAULT '{}'::character varying[] NOT NULL,
```

Comment:
Does the `portfolio` table need `administrators`, `contributors` and `readOnlyOperators`? And does `administrators` equate to `portfolioAdministrators` as shown in the new internal API Spec? I am only asking since  only `portfolioAdministrators` was available at the top level of the portfolio.

Reply:
Good eye.  `administrators` does equate to `portfolioAdministrators`.  `contributors` and `readOnlyOperators` are not necessary per spec, but present due to entity inheritance from `ProvisionableEntity`.  This was by design, but a potential area of improvement in the entity modeling.

Reply:
Thanks for clearing that up.